assets/src/scss/editorOverrides.scss

```diff
@@ -1,4 +1,5 @@
 @import "./base/tokens";
+@import "./core-overrides/image";
```

Comment:
Maybe it would be better to import this in the [editorStyle](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/editorStyle.scss) file with the other block styles? 🤔